temperatures/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, DateTime, Float, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DbTemperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime, nullable=False)
+    temperature = Column(Float, nullable=False)
+    city_id = Column(Integer, ForeignKey("cities.city_id"))
```

Comment:
The ForeignKey reference should match the actual table name and column name in the 'cities' table. Ensure that 'cities.city_id' is correct and matches the 'DbCity' model definition.